keps/sig-network/1435-mixed-protocol-lb/README.md

```diff
@@ -487,6 +487,39 @@ From CPI implementation perspective thet feature can be graduated to beta, as th
 
 Graduating to GA means, that the feature flag checking is removed from the code. It means, that all CPI implementations must be ready to deal with Services with mixed protocol configuration - either rejecting such Services properly or managing the cloud load balancers according to the Service definition.
 
+#### Alpha Graduation
+
+- Feature is implemented and controller with a feature flag. The feature flag is disabled by default.
+
+#### Alpha -> Beta Graduation
+
+- Gather feedback from developers and surveys
```

Comment:
Specifically I'd say we need to prove it works correctly and that it doesn't-work correctly (errors).

Reply:
That was part of the template. Fixed.